<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebediaClass</class>
 <widget class="QMainWindow" name="WebediaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Webedia</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>160</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_creer_scene">
       <property name="text">
        <string>Création d'une scène :</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_nom">
         <property name="text">
          <string>Nom :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_nom_scene"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_creer_scene">
       <property name="text">
        <string>Créer la scène</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_bdd">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>721</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_console">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>160</width>
      <height>168</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Liste Scène BDD :</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>1 :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>2 :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>3 :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>4 :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Afficher les Scènes</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCr_ation">
   <property name="text">
    <string>Ajout</string>
   </property>
  </action>
  <action name="actionModifier">
   <property name="text">
    <string>Modifier</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionCr_ation_2">
   <property name="text">
    <string>Création</string>
   </property>
  </action>
  <action name="actionModifier_2">
   <property name="text">
    <string>Modifier</string>
   </property>
  </action>
  <action name="actionSupprimer_2">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionAffiche">
   <property name="text">
    <string>Affiche</string>
   </property>
  </action>
  <action name="actionAffiche_2">
   <property name="text">
    <string>Affiche</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Webedia.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_creer_scene</sender>
   <signal>clicked()</signal>
   <receiver>WebediaClass</receiver>
   <slot>onButtonClickedScene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonClickedScene()</slot>
 </slots>
</ui>
